versions/datastax/3.24.0/ignore.yaml

```diff
@@ -17,6 +17,33 @@ tests:
     - tests.integration.standard.test_query.LightweightTransactionTests.test_was_applied_batch_stmt
     - tests.integration.standard.test_types.TypeTests.test_can_read_composite_type
   flaky:
+    # def test_deprecation_warnings_legacy_parameters(self):
+    #        """
+    #        Tests the deprecation warning has been added when using
+    #        legacy parameters
+    #
+    #        @since 3.13
+    #        @jira_ticket PYTHON-877
+    #        @expected_result the deprecation warning is emitted
+    #
+    #        @test_category logs
+    #        """
+    #        with warnings.catch_warnings(record=True) as w:
+    #            TestCluster(load_balancing_policy=RoundRobinPolicy())
+    #>           self.assertEqual(len(w), 1)
+    #E           AssertionError: 2 != 1
+    #
+    #tests/integration/standard/test_cluster.py:1530: AssertionError
```

Comment:
@Orenef11 why do we need these comments here?
I don't think it's the place.
If you want to explain the reason, you can use the commit message.

Reply:
Done